pkg/ui/handler.go

```diff
@@ -166,8 +154,14 @@ func (u *Handler) IndexFile() http.Handler {
 }
 
 func serveIndex(resp io.Writer, url string) error {
-	r, err := insecureClient.Get(url)
+	client := &http.Client{
+		Transport: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+		},
+	}
+	r, err := client.Get(url)
 	if err != nil {
+		logrus.Errorf("failed to download %s: %v", url, err)
```

Comment:
We already log the error here: https://github.com/rancher/steve/blob/da9bc2a11474d661f7c7213ff03d8896360eabce/pkg/ui/handler.go#L112 so instead of logging it twice, maybe best to just return it?

The error is ignored here: https://github.com/rancher/steve/blob/da9bc2a11474d661f7c7213ff03d8896360eabce/pkg/ui/handler.go#L161, we could add a logging statement there instead.